ion_functions/data/perf/test_adcp_performance.py

```diff
@@ -25,19 +25,23 @@ class TestADCPPerformance(PerformanceTestCase):
 
     def setUp(self):
         # set test inputs -- values from DPS
-        self.b1 = np.array([-0.0300, -0.2950, -0.5140, -0.2340, -0.1880,
-                            0.2030, -0.3250,  0.3050, -0.2040, -0.2940]) * 1000
-        self.b2 = np.array([0.1800, -0.1320,  0.2130,  0.3090,  0.2910,
-                            0.0490,  0.1880,  0.3730, -0.0020, 0.1720]) * 1000
-        self.b3 = np.array([-0.3980, -0.4360, -0.1310, -0.4730, -0.4430,
-                            0.1880, -0.1680,  0.2910, -0.1790, 0.0080]) * 1000
-        self.b4 = np.array([-0.2160, -0.6050, -0.0920, -0.0580,  0.4840,
-                            -0.0050,  0.3380,  0.1750, -0.0800, -0.5490]) * 1000
-        self.echo = np.array([0, 25, 50, 75, 100, 125, 150, 175, 200, 225, 250])
+        # set test inputs -- values from DPS (reset to integers, matches format from instrument)
+        self.b1 = np.array([[-30, -295, -514, -234, -188, 203, -325, 305, -204, -294]])
+        self.b2 = np.array([[180, -132, 213, 309, 291, 49, 188, 373, 2, 172]])
+        self.b3 = np.array([[-398, -436, -131, -473, -443, 188, -168, 291, -179, 8]])
+        self.b4 = np.array([[-216, -605, -92, -58, 484, -5, 338, 175, -80, -549]])
+
+        self.pg1 = np.array([[100, 100, 100,  24, 100, 100,  24, 100, 100, 100]])
```

Comment:
Should we include some values of 25 or 26 in these percent good arrays just to exercise the other side of the percent good floor (with something closer to the threshold than 100)?

Reply:
Mixed up the percent good array to include a bunch of other values. Hopefully that will work.